libraries/AP_HAL_ChibiOS/hwdef/JFB100/hwdef.dat

```diff
@@ -0,0 +1,355 @@
+# hw definition file for processing by chibios_hwdef.py
+# for JAE JFB-100 hardware.
+define FW_COMMENT "(JFB100 V1.0.0)"
+
+# MCU class and specific type. It is a F765, which is the same as F767
+# but without the TFT interface
+MCU STM32F7xx STM32F767xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+define CONFIG_HAL_BOARD_SUBTYPE HAL_BOARD_SUBTYPE_CHIBIOS_FMUV5
+define HAL_CHIBIOS_ARCH_FMUV5 1
```

Comment:
It doesn't look like a FMUv5 board, you shouldn't need these lines

Reply:
JFB100 is using fmu-v5 architecture.